cmd/nodecmd/create.go

```diff
@@ -335,6 +335,7 @@ func createClusterNodeConfig(network models.Network, cloudConfigMap models.Cloud
 				SecurityGroup: cloudConfig.SecurityGroup,
 				ElasticIP:     publicIP,
 				CloudService:  cloudService,
+				HasDynamicIP:  !useStaticIP,
```

Comment:
rename `HasDynamicIP` to `useStaticIP` or `StaticIP` and get rid of bunch of  `!`  below :)